pages/common/keepassxc-cli.md

```diff
@@ -0,0 +1,36 @@
+# keepassxc-cli
+
+> Command-line interface for KeepassXC.
+> More information: <https://keepassxc.org/>.
+
+- Search entries:
+
+`keepassxc-cli lookup {{/path/to/database_file}} {{name}}`
+
+- List the contents a folder:
+
+`keepassxc-cli ls {{/path/to/database_file}} {{folder_name}}`
```

Comment:
Should this be a path to a folder or just the folder name?

Reply:
The folder name, except if this is nested, but usually it is not nested